filter.js

```diff
@@ -27,9 +29,11 @@ class Filter {
      */
     if (filterValues[0] === true || filterValues[1] === true) {
       filterValues[2] = true
+      document.getElementById("filter-both").checked = true
+      document.getElementById("filter-both").disabled = true
     }
-    if (!filterValues[0] && !filterValues[1]) {
-      filterValues[2] = false
+    else {
+      document.getElementById("filter-both").disabled = false
```

Comment:
I haven't gotten the chance to get this pulled locally, just want to verify my understanding of under what condition this check is being enabled. Looks like if filterValues[1] or filterValues[0] are not checked, the filter would be enabled?

Reply:
@mc-funk yeah if filterValues[0] ("receiving") and filterValues[1] ("distributing") are _both_ unchecked, then "open for both" is enabled to be toggled on or off